models/comments-model.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../db/connection")
+
+
+exports.fetchCommentsByreviewID = (review_id) => {
+    return db
+      .query(`
+    SELECT comments.*
+    FROM comments
+    LEFT JOIN users ON comments.author = users.username
+    WHERE comments.review_id = $1
+    ORDER BY created_at DESC;`
+    ,[review_id])
+        .then(({ rows }) => {
+        if (rows.length === 0) {
+          return Promise.reject({ status: 404, msg: 'ID NOT FOUND' });
```

Comment:
this will error for an id not found if the review exists, but has no comments, which isn't what we want